addons/html_builder/static/src/website_builder/plugins/form/form_field_option.js

```diff
@@ -19,7 +21,11 @@ export class FormFieldOption extends BaseOptionComponent {
             conditionValueList: [],
             dependencyEl: null,
         });
-        this.domState = useDomState((el) => ({ el }));
+        this.domState = useDomState((el) => {
+            const modelName = el.closest("form")?.dataset.model_name;
```

Comment:
It s possible to have an el outside of a <form> for this option ?

Reply:
Users might destroy their HTML, so there is no guarantee it will be there. And it that case, I'd rather avoid potential blocking crashes.